media/static-bundles.json

```diff
@@ -1487,6 +1490,9 @@
     },
     {
       "files": [
+        "js/base/mozilla-fxa.js",
```

Comment:
I don't think we need this file to determine signed in state, is that right?

Reply:
Nope, we're doing anything to detect state, or at least I wasn't given any guidance on what to show to signed-in users, and since it's firstrun it's unlikely folks seeing it would already be signed in. So I'll remove the state detection bits.